maxim/tests/test_simulation.py

```
@@ -0,0 +1,802 @@
+        self.assertIsNotNone(result)
+        self.assertIsNotNone(result.test_run_result)
+        self.assertIsNotNone(result.test_run_result.link)
+        print(f"✅ {test_case}: {result.test_run_result.link}")
+
+    # ----- Invalid Combinations - Error Handling -----
+
+    def test_invalid_no_prompt_or_workflow(self) -> None:
+        test_case = "invalid-no-prompt-or-workflow"
+        logger = TestLogger(test_case)
+        logger.info("Starting test, Invalid Combinations")
+        with self.assertRaises(Exception):
```

Ruff (0.14.14)

Do not assert blind exception: `Exception`

(B017)